test/karma/helpers/ParentClickView.js

```diff
@@ -0,0 +1,35 @@
+var _ = require('underscore');
+var TorsoView = require('./../../../modules/View');
+var ChildClickView = require('./ChildClickView');
+
+module.exports = TorsoView.extend({
+  events: {
+    'click div.parent' : 'myClick'
+  },
+  initialize: function() {
+    this.spyHook();
+
+    spyOn(this, 'myClick');
+    spyOn(this, 'afterMyEvent');
+    spyOn(this, 'afterMyDeactivatableEvent');
```

Comment:
It sounds like to me, the reason you don't want spyOn to appear in the View's initialize is because people would look at the test classes as an example for how to instantiate real Torso views. I tried to only use spyHook to attach listeners, but it got messy with ParentClickView because the ChildClickViews are being instantiated inside ParentClickView.initialize. Would it be acceptable to you if we indicated that the views are instrumented for testing by naming them Spied*View.js?